bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/checksum/CRC32CDigestManager.java

```diff
@@ -38,10 +39,10 @@ protected MutableInt initialValue() throws Exception {
         }
     };
 
-    public CRC32CDigestManager(long ledgerId, boolean useV2Protocol) {
-        super(ledgerId, useV2Protocol);
+    public CRC32CDigestManager(long ledgerId, boolean useV2Protocol, ByteBufAllocator allocator) {
+        super(ledgerId, useV2Protocol, allocator);
         if (!Sse42Crc32C.isSupported()) {
-            log.error("Sse42Crc32C is not supported, will use less slower CRC32C implementation.");
+            log.error("Sse42Crc32C is not supported, will use slower CRC32C implementation.");
```

Comment:
If you're correcting the grammar, you should at least include the indefinite article ;)

Let's see if this suggestion thing works...

```suggestion
            log.error("Sse42Crc32C is not supported, will use a slower CRC32C implementation.");
```